components/Archetype/interfaces.rst

```diff
@@ -0,0 +1,20 @@
+Interfaces
+==========
+
+.. note::
+
+    For more detailed information about the interfaces go to `Sylius Api`_.
```

Comment:
I think, that straightforward links to interfaces (which you have below) are enough :)